.vscode/launch.json

```diff
@@ -9,9 +9,18 @@
             "type": "lua-local",
             "request": "launch",
             "program": {
-                "lua": "lua5.1",
+                "lua": "lua",
```

Comment:
This should probably remain lua5.1. This is so users may have multiple lua installs on their machine (especially since lua 5.1 is so old).

Reply:
OK

Reply:
Done